src/style/core.scss

```diff
@@ -93,6 +93,23 @@ h1 code {
     font-weight: 400;
 }
 
+#content a {
+    color: var(--f-text-color-default);
```

Comment:
Det är nog bra om vi lägger in en ny variabel i [`css-variables.mjs`](https://github.com/Forsakringskassan/docs-generator/blob/main/src/style/css-variables.mjs) istället så vi inte bygger in / tvingar FKUI här.

Nuvarande variabler kan visas under [Theme](https://forsakringskassan.github.io/docs-generator/theme/) i menyn, det finns några nya som inte är släppta än.